tests/suites/cmr/offer_consume.sh

```diff
@@ -0,0 +1,134 @@
+# Exercising the CMR Juju functionality which allows applications to
+# communicate between different models.
+# This test will exercise the following aspects:
+#  - Ensure a user is able to create an offer of an applications' endpoint
+#    including:
+#      - A user is able to consume and relate to the offer
+#      - Workload data successfully provided
+#      - The offer appears in the list-offers output
+#      - The user is able to name the offer
+#      - The user is able to remove the offer
```

Comment:
we're not testing all of these things in this particular file though?

Reply:
I've added `list-offers` and `find-offers` checks. And offer naming, so looks like we now check all these points.